### ISSUE TITLE
File Overwrite Warning on First Install of Portable App

### ISSUE THREAD

**Issue by Trenly** [microsoft/winget-cli#2327]
### Brief description of your issue

When first installing a Portable App, there is a warning displayed that a file is being overwritten, even though it is a clean install (running in windows sandbox)

### Steps to reproduce

Unzip the manifest attached; 
Install the manifest locally using v1.3 of the winget client

Manifest Files: [OO-Software.zip](https://github.com/microsoft/winget-cli/files/9093714/OO-Software.zip)
Log Files: [OO-Software.ShutUp10.zip](https://github.com/microsoft/winget-cli/files/9093744/Logs.for.OO-Software.ShutUp10.zip)

### Expected behavior

No Warning to be shown, as that file shouldn't exist from a clean sandbox install.
<details>
<summary>Expected Command Line Output from Install</summary>

```
--> Installing WinGet
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Admin setting enabled.

--> Installing the Manifest 1.9.1431.368

Found ShutUp10++ [OO-Software.ShutUp10] Version 1.9.1431.368
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl5.oo-software.com/files/ooshutup10/OOSU10.exe
  ██████████████████████████████  1.58 MB / 1.58 MB
Successfully verified installer hash
Starting package install...
Path environment variable modified; restart your shell to use the new value.
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName    DisplayVersion Publisher    ProductCode
-----------    -------------- ---------    -----------
O&O ShutUp10++ 1.9.1431       O&O Software OO-Software.ShutUp10__DefaultSource
```
</details>

### Actual behavior

There is a warning shown - 
> Overwriting existing file:C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\WinGet\Links\shutup10.exe
<details>
<summary>Command Line Output from Install</summary>

```
--> Installing WinGet
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Admin setting enabled.

--> Installing the Manifest 1.9.1431.368

Found ShutUp10++ [OO-Software.ShutUp10] Version 1.9.1431.368
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl5.oo-software.com/files/ooshutup10/OOSU10.exe
  ██████████████████████████████  1.58 MB / 1.58 MB
Successfully verified installer hash
Starting package install...
Overwriting existing file:C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\WinGet\Links\shutup10.exe
Path environment variable modified; restart your shell to use the new value.
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName    DisplayVersion Publisher    ProductCode
-----------    -------------- ---------    -----------
O&O ShutUp10++ 1.9.1431       O&O Software OO-Software.ShutUp10__DefaultSource
```

</details>
<details>
<summary>Screen Capture from Install</summary>

![image](https://user-images.githubusercontent.com/12611259/178512490-acbfad50-65b7-4390-b869-e0a6f32ee3fd.png)

</details>

### Environment

```shell
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget --info
Windows Package Manager v1.3.1872
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19041.1766
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.1872.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale
```

**Comment by Trenly** [comment]
If more manifests are needed for testing - There are extras in the latest commits on these branches
https://github.com/Trenly/winget-pkgs/branches/all?query=Portables

**Comment by denelon** [comment]
I think we will do a servicing fix to get this into 1.3 after GA, and make sure it gets into the next 1.4 preview.
